src/data/valid/Database-gen_from_omics.yaml

```
@@ -0,0 +1,28 @@
+data_object_set:
+- id: nmdc:dobj-11-dtTMNb
+  data_object_type: Metagenome Raw Reads
+  description: "my awesome fastq file"
+  file_size_bytes: 1382304
+  md5_checksum: "22afa3d49b73eaec2e9787a6b88fbdc3"
+  name: "1237.14.3718.ATCG-ACTG.fastq.gz"
+  type: "nmdc:DataObject"
+  was_generated_by: "nmdc:omprc-99-123456"
+omics_processing_set:  
+ - id: nmdc:omprc-99-123456
+   alternative_identifiers:
+    - gold:Gp0108335
+   name:  Permafrost 712P3D
```

In this comment, I am suggesting the removal of a whitespace character I assume was included unintentionally.

```suggestion
   name:  Permafrost 712P3D
   name: Permafrost 712P3D   
```